homeassistant/util/yaml.py

```diff
@@ -64,6 +65,25 @@ def _ordered_dict(loader, node):
 
     return OrderedDict(nodes)
 
+
+def _env_var_yaml(loader, node):
+    """Load environment variables and embed it into the configuration YAML."""
+    pattern = re.compile(r'\<%= ENV\[\'(.*)\'\] %\>')
+    value = loader.construct_scalar(node)
+    env_var = pattern.match(value).group(1)
+
+    try:
+        return os.environ[env_var]
+    except Exception as exc:
+        _LOGGER.error("Environment variable %s not defined.", exc)
+        raise HomeAssistantError(exc)
+
+# Look for any environment variable tags and convert it to the host variable
+yaml.SafeLoader.add_implicit_resolver('!env_var',
```

Comment:
Instead of adding this, let's just stick to the normal constructor `!env_var`. No need to wrap all the strings in some unnecessary syntax.